### ISSUE TITLE
macOS - "Please enable ARC (Automatic Reference Counting) at the project level"

### ISSUE THREAD

**Issue by dimitre** [openframeworks/openFrameworks#8060]
Using all the latest changes in OF brings this warning:
"Please enable ARC (Automatic Reference Counting) at the project level"
<img width="1037" alt="Screenshot 2024-07-28 at 15 10 09" src="https://github.com/user-attachments/assets/113cb871-6822-474c-88d3-ed5b316784fd">

**Comment by danoli3** [comment]
Yeah it was an Error before if had any objective-c++ and -fno-arc on a file
